client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-details.tsx

```diff
@@ -31,6 +35,9 @@ export const UpgradePlanDetails = ( props: Props ) => {
 	const plan = getPlan( selectedPlan );
 	const planId = plan?.getProductId();
 
+	const { list: upgradePlanHostingDetailsList, isFetching: isFetchingHostingDetails } =
```

Comment:
It's done in the parent component now to control the fetching along with the others.